CP03ISSM/CP03ISSM_R00009_ingest.csv

```diff
@@ -0,0 +1,43 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/cpm3/syslog/cpm_status*.txt,CP03ISSM-MFC31-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/syslog/*.syslog.log,CP03ISSM-MFD35-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd8.adcpt_acfgm_dcl_pd8_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/adcp/*adcp.log,CP03ISSM-MFD35-01-ADCPTF000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/instruments/DCL36/ADCP_sn*/_RDI*.000,CP03ISSM-MFD35-01-ADCPTF000,recovered_inst,Available,
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/presf/*presf.log,CP03ISSM-MFD35-02-PRESFB000,recovered_host,Available,
+mi.dataset.driver.presf_abc.presf_abc_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/instruments/DCL36/PRESF_sn*/PRESF*.hex,CP03ISSM-MFD35-02-PRESFB000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/velpt2/*velpt2.log,CP03ISSM-MFD35-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/instruments/DCL36/VELPT_snAQD*/*.aqd,CP03ISSM-MFD35-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/pco2w/*pco2w.log,CP03ISSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/instruments/DCL36/PCO2W_snC*/SAMI*.txt,CP03ISSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/cg_data/DCL36/phsen2/*phsen2.log,CP03ISSM-MFD35-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00009/instruments/DCL36/PHSEN_sn*/SAMI*.txt,CP03ISSM-MFD35-06-PHSEND000,recovered_inst,Available,

```

Comment:
A number of these have DCL36 in the filemask instead of dcl36.  I don't know if matters or not.  In the Raw Data Repo, the folders are all small letters.